.github/workflows/report.yml

```diff
@@ -19,7 +19,29 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  download-prev-reports:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download previous test report from local artifact
+        uses: dawidd6/action-download-artifact@v3
+        with:
+          workflow: report.yml
+          branch: test-workflow
```

Comment:
This needs removing now.

Reply:
Ah my bad, was using it for testing, forgot to change